src/helpers/errorHelpers.js

```diff
@@ -0,0 +1,15 @@
+const errorHelpers = {
```

Comment:
I think using the response generator helper here will be helpful instead of creating a different abstraction for 404 errors.

Reply:
Noted